pycbc/waveform/compress.py

```diff
@@ -653,21 +704,23 @@ def decompress(self, out=None, df=None, f_lower=None, interpolation=None):
             The decompressed waveform.
         """
         if f_lower is None:
-            # use the minimum of the samlpe points
-            f_lower = self.sample_points.min()
+            # use the minimum of the sample points
+            f_lower = numpy.array([self.amplitude_freq_points.min(),
+                       self.phase_freq_points.min()]).max()
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->